images/cluster-setup/Dockerfile

```diff
@@ -22,7 +22,8 @@ RUN KUBE_VERSION=v1.24.0 && \
     curl -L -o /usr/local/bin/kubectl "https://dl.k8s.io/release/$KUBE_VERSION/bin/linux/amd64/kubectl" && \
     chmod 755 /usr/local/bin/kubectl
 RUN microdnf install -y findutils-4.6.0 git-2.31.1 && microdnf clean all
-COPY ./install.sh /usr/local/bin/install.sh
+COPY bin /usr/local/bin
+RUN chmod 755 /usr/local/bin/*.sh
```

Comment:
Could you please be specific and apply the chmod to the files you are adding?
I don't think there is any shell script in the base image but if one gets added I would not like this Dockerfile to change its access rights unknowingly.

Reply:
fixed